tests/functional/firefox/test_home.py

```diff
@@ -16,7 +16,9 @@ def test_download_menu_list_displays(base_url, selenium):
 
 
 @pytest.mark.nondestructive
-@pytest.mark.skip_if_not_firefox(reason='FB Container link shown only to Firefox users.')
-def test_fb_container_fx(base_url, selenium):
+def test_sticky_promo(base_url, selenium):
```

Comment:
There was not a sticky promo test on /firefox, so I added it here.

Reply:
I pushed this as it but received an error when running the test locally: 

```
selenium.common.exceptions.ElementClickInterceptedException: 
Message: Element <button class="mzp-c-sticky-promo-close" type="button"> is not clickable at point (1266,407) because another element <html class="osx x86 js is-firefox is-modern-browser mzp-has-sticky-navigation loaded"> obscures it
```

Looking at the screenshot from the generated report, it doesn't look obscured. Could you please help me debug? 

![image](https://user-images.githubusercontent.com/2692333/106947553-242c4f80-66f0-11eb-9431-c9fcccdba61b.png)